server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SelectContestsForAudit.java

```diff
@@ -71,18 +70,18 @@ public synchronized String endpoint(final Request the_request,
     try {
       final ContestToAudit[] contests = 
           Main.GSON.fromJson(the_request.body(), ContestToAudit[].class);
-      final DoSDashboard dosdb = DoSDashboardQueries.get();
+      final DoSDashboard dosdb = Persistence.getByID(DoSDashboard.ID, DoSDashboard.class);
       if (dosdb == null) {
         Main.LOGGER.error("could not get department of state dashboard");
         serverError(the_response, "Could not select contests");
       } else {
         for (final ContestToAudit c : contests) {
           Main.LOGGER.info("updating contest audit status: " + c);
           dosdb.updateContestToAudit(c);
+          Persistence.saveOrUpdate(dosdb);
```

Comment:
No, I don't think so. And in fact, a future PR is probably going to rip out a lot of those persistence calls in favor of letting Hibernate just do its job.